### ISSUE TITLE
ModuleNotFoundError: No module named 'typing_extensions'

### ISSUE THREAD

**Issue by a0g83agbc84** [gtronset/beets-filetote#145]
## Problem

When running `beet import` I get the following error:

```
** error loading plugin filetote:
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/beets/plugins.py", line 268, in load_plugins
    namespace = __import__(modname, None, None)
  File "/usr/local/lib/python3.9/site-packages/beetsplug/filetote.py", line 16, in <module>
    from .filetote_dataclasses import (
  File "/usr/local/lib/python3.9/site-packages/beetsplug/filetote_dataclasses.py", line 22, in <module>
    from typing_extensions import TypeAlias  # pylint: disable=import-error
ModuleNotFoundError: No module named 'typing_extensions'
```

My expectation was:
___

Plugin to be loaded correctly.

* OS: Docker compose / python:3.9-alpine
* Python version: 3.9.18
* beets version: 1.6.0
* Filetote version: 0.4.7 (installed using pip)
* Turning off other plugins made problem go away (yes/no): no

**Comment by gtronset** [comment]
@a0g83agbc84 thank you for reporting this! This is occurring due to the module [typing_extensions](https://pypi.org/project/typing-extensions/) not being installed. Installing it should remove this error. That said, I'm working to remove this dependency as its value for users isn't worth the setup. This will be changed in the next release so that this dependency is no longer needed.

**Comment by a0g83agbc84** [comment]
Thank you very much for your answer. I'll install that module and go with that for now, but looking forward to updating to the version that does not need this module. Looking forward to trying your plugin!

**Comment by gtronset** [comment]
This should now be resolved in [v0.4.8](https://github.com/gtronset/beets-filetote/releases/tag/v0.4.8)!
